src/storage/AddEdgesProcessor.cpp

```diff
@@ -16,12 +16,18 @@ void AddEdgesProcessor::process(const cpp2::AddEdgesRequest& req) {
     auto spaceId = req.get_space_id();
     auto version =
         std::numeric_limits<int64_t>::max() - time::WallClock::fastNowInMicroSec();
+    // Switch version to big-endian, make sure the key is in ordered.
+    version = folly::Endian::big(version);
```

Comment:
can we use PC's time to version in Distributed Systems?

Reply:
Use `max - timestamp` is to make sure the version is in descending order and the latest version is the first one when you scan rocksdb with prefix.